structures/structures-02.lsp

```diff
@@ -378,12 +378,12 @@
 
 (eval-when (:load-toplevel :execute)
   (let ((x nil))
-    (flet ((%f () x)
-          (%g (y) (setf x y)))
+    (flet ((OOf () x)
+          (OOg (y) (setf x y)))
```

Comment:
I believe you are having troubles here because in eus the character `%` is bound to the infix macro.

However, we deactivate this macro in https://github.com/euslisp/ansi-test/blob/gsoc2018/eus-test.l#L102, so it should be fine to do `(flet ((%f () x))` if loaded from `eus-test.l`.

Reply:
That's right, I've overlooked that. Now, do you think we should get rid of the infix macro for common lisp compliance? Personally, I'm of the opinion it might be advantageous to keep it, and change the other %s that may appear.. it is non compliant, but would require only a simple, quick, almost painless change in the code, and would keep the infix macro (which seems useful). Another option would be binding the infix macro to another thing, but I don't know of the codes that makes use of it by now and how they could react (not too gently, I guess)

Reply:
The euslisp infix is barely used, probably because almost nobody actually knows about it. We have some hits on eus source code and `lib/`, but I would say that it is pretty much safe for other codes (with some exceptions, of course). On the other side, having a look at this `ansi-test` package and in the source code for `sbcl` and `clisp`, I would say `%` is quite used for naming auxiliary functions. 

We have no good reason to delete the infix, but I would suggest a naming change if we happen to create a `common lisp compliant branch` in the eus repository, since there seems to be more uses outside eus than inside it. If should definitely be kept this way for master branch, of course.

Infix notation in common lisp tends to be named (not surprisingly) as `#i`. The problem is that we use this for integers arrays on eus (which in my opinion is also quite misleading, seems it makes me think of imaginary numbers).
https://github.com/rigetticomputing/cmu-infix
https://www.cs.cmu.edu/Groups/AI/lang/lisp/code/syntax/infix/infix.cl

Anyways, since we want to keep our test files close to the original ones, I would suggest leaving the `%` for now.